cos-fleetshard-operator-debezium/src/main/java/org/bf2/cos/fleetshard/operator/debezium/DebeziumOperandController.java

```diff
@@ -218,6 +220,13 @@ protected List<HasMetadata> doReify(
                     connectorConfig.put(CONFIG_OPTION_POSTGRES_PLUGIN_NAME, PLUGIN_NAME_PGOUTPUT);
                 }
                 break;
+            case CLASS_NAME_SQLSERVER_CONNECTOR:
```

Comment:
@ani-sha  @Naros This is the right place to handle connector specific additional configuration.

Reply:
Thanks for pointing it out.